atomic_defi_design/qml/Portfolio/Portfolio.qml

```diff
@@ -224,12 +224,12 @@ Item {
                     anchors.verticalCenter: parent.verticalCenter

```

Comment:
if you want to propose GUI changes - please create separate PR to not mix it with translation

Reply:
there were just 2 or 3 buttons that needed a fix, and then I could not stop translating, you are right, will definitely keep in mind for future contributions though. however translation and button fixes lie in separate commits if this helps somehow.

Reply:
as long as it's fine for @Milerius